packages/apputils/src/iframe.ts

```diff
@@ -36,6 +96,7 @@ namespace Private {
   export function createNode(): HTMLElement {
     let node = document.createElement('div');
     let iframe = document.createElement('iframe');
+    iframe.setAttribute('sandbox', '');
```

Comment:
Cool. Empty is almost brokenly "secure," but we want developers to opt their users into this thing only with care.

Let's keep going with the security model.

Setting the [`referrerPolicy`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/referrerPolicy) to the "annoyingly secure by default," `no-referrer` and again offer an enumeration of other possible values seems like a good way forward.

Doesn't work on IE/Edge, but meh.

Looks like [`csp`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/csp) isn't ready for primetime.

Reply:
Sure, that makes sense to me. I'll follow basically the same model.